src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantDataManager.java

```diff
@@ -362,9 +367,11 @@ private static double decodeAnnotation( final String annotationKey, final Varian
                     value += vrac.MQ_JITTER * Utils.getRandomGenerator().nextGaussian();
                 }
             }
-        } catch( Exception e ) {
-            //TODO: what exception is this handling ? it seems overly broad
-            value = Double.NaN; // The VQSR works with missing data by marginalizing over the missing dimension when evaluating the Gaussian mixture model
+            if( jitter && (annotationKey.equalsIgnoreCase(GATKVCFConstants.AS_RMS_MAPPING_QUALITY_KEY))){
+                value += vrac.MQ_JITTER * Utils.getRandomGenerator().nextGaussian();
+            }
```

Comment:
Is it worth updating the MQ_CAP doc (to explicitly state that its MQ only and doesn't affect AS_MQ) ? Up to you. It might actually be more correct now.